internal/provider/test_constants.go

```diff
@@ -65,6 +65,29 @@ resource "aci_vrf" "test" {
   name      = "test_vrf"
 }
 `
+const testConfigFvAEPgMinDependencyWithFvTenant = testConfigFvAEPgMin
+
+const testConfigFvSiteAssociatedMinDependencyWithFvCtx = testConfigFvTenantMin + `
+resource "aci_vrf" "test" {
+  tenant_dn = aci_tenant.test.id
+  name      = "test_vrf"
+}
+
+resource "aci_associated_site" "test" {
+  parent_dn = aci_vrf.test.id
+  site_id = "102"
+  name   = "test_associated_site"
+}
+`
```

Comment:
think these should not be needed and will need to be auto-generated when the parents: are correctly defined in the properties files.